src/pages/EnablePayments/ActivateStep.js

```diff
@@ -35,6 +40,13 @@ class ActivateStep extends React.Component {
     }
 
     renderGoldWalletActivationStep() {
+        let continueButtonText = this.props.translate('common.continue');
+        if (this.props.userWallet.sourceAction) {
+            continueButtonText = this.props.userWallet.sourceAction === 'transferBalance'
```

Comment:
`transferBalance` can be a constant too

Reply:
(Same question as with `sourceAction="iou"`)